checks/diskspace/diskspace_test.go

```diff
@@ -0,0 +1,50 @@
+package diskspace
+
+import (
+	"context"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+// Set up with threshold of maximum value of uint64 and expect no failure
+func TestExternalCheckWithMaximumThreshold(t *testing.T) {
+	check := New(^uint64(0), "/")
```

Comment:
Instead of using the caret operator, maybe we should just use `math.MaxUint64`?

Reply:
My mistake for not, I am new(ish) to Go. That is indeed much better and I will update it.